compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/gestures/UikitScrollable.uikit.kt

```diff
@@ -27,9 +28,22 @@ import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.dp
 
 @Composable
-internal actual fun platformScrollConfig(): ScrollConfig = UiKitConfig
+internal actual fun platformScrollConfig(): ScrollConfig = UiKitScrollConfig
+
+/**
+ * Opt out of the Cupertino overscroll behavior (rubber banding and spring effect).
+ *
+ * This method should be called before any @Composable function using this effect is executed
+ * (so as early as possible, e.g. during app start up).
+ */
+@OptIn(ExperimentalFoundationApi::class)
```

Comment:
`OptIn` is about enabling using of experimental API, but not marking it as it

Reply:
```suggestion
@ExperimentalFoundationApi
```

Reply:
Fixed